react/components/BulkImportList/BulkImportList.tsx

```diff
@@ -1,27 +1,22 @@
-import React from 'react'
-import { ImportAlertError } from '@vtex/bulk-import-ui'
+import React, { useState } from 'react'
 
-import { useBulkImportsQuery, useTranslate } from '../../hooks'
+import { useBulkImportsQuery } from '../../hooks'
 import ImportAlertList from '../ImportAlertList/ImportAlertList'
 
 const BulkImportList = () => {
-  const { data, error, mutate } = useBulkImportsQuery()
-
-  const { translate: t } = useTranslate()
+  const [shouldPoll, setShouldPoll] = useState(true)
+  const { data, error } = useBulkImportsQuery({
```

Comment:
This hook is called here and in the [CreateOrganizationButton](https://github.com/vtex-apps/b2b-organizations/pull/148/files#diff-9f9587e581c3d3a0ab9a3bb560d0675f0e46dd8ba67794bded35b168badd7cfaR36), the SWR caches the response and share the data in both contexts